src/reservoir/Reservoir.ts

```diff
@@ -82,26 +91,40 @@ export class Reservoir {
     });
   }
 
-  async getAsk({ orderId }: { orderId: Hash }) {
+  async getAsk({ orderId }: { orderId: string }) {
     return fetch(
       `${this.baseApiUrl}/orders/asks/v5?ids=${orderId}&includeRawData=true`
     )
-      .then((res) => res.json() as Promise<{ orders: unknown[] }>)
       .then(
-        ({ orders }) =>
-          orders[0] as paths["/asks/v5"]["get"]["responses"]["200"]["schema"]
-      );
+        (res) =>
+          res.json() as Promise<{
+            orders: paths["/orders/asks/v5"]["get"]["responses"]["200"]["schema"]["orders"];
+          }>
+      )
+      .then(({ orders }) => {
+        if (!orders) {
+          throw new Error("");
```

Comment:
Let's keep the error from https://github.com/gondixyz/gondi-js/pull/27/files#diff-fa0cb785ea9a97898c256a6b8195673c5b8c4f9a844fc317ee2de0f556caabf5R93